### ISSUE TITLE
Allow Custom Primary Color in the `Global` Settings

### ISSUE THREAD

**Issue by sherifkayad** [kubernetes/dashboard#6654]
##### What would you like to be added

As a follow-up on #2618 and since the dashboard now supports Themes from the user-specific settings, I was thinking that it would be really of benefit if we could have the `primary` color externalized and available as a **global** setting that can be maintained in the settings ConfigMap.

Maybe the ConfigMap introduces a new property e.g. `pirmaryColor` and the value could be the Hex color code ..

In my view such a setting should NOT be user specific (rather a global setting) on an instance level.

##### Why is this needed
Users with different Kubernetes Clusters (e.g. Non PROD / PROD / LAB / .. etc.) would love to distinguish (optically) between these clusters and the users of the Kubernetes Dashboards in each would like an indication in which cluster they are at the moment .. Assuming the Cluster admin installs and configured the k8s Dashboard in each cluster and configures a unique color for each; it would be visually possible for the Dashboard users to distinguish between the different clusters in an easy way.

For example:
- PROD Cluster(s) would get a Red Header / Primary Color
- Non-PROD Cluster(s) would get a Green Header / Primary Color
- LAB might get Purple or whatever

##### Comments

My dumb imagination for e.g. the k8s Dashboard in the PROD Cluster:
![image](https://user-images.githubusercontent.com/8362784/145058308-6d6bb3f4-a774-48c6-b611-ff38f691dd81.png)

**Comment by sherifkayad** [comment]
@floreks any updates for this one here?

**Comment by floreks** [comment]
I don't think we want to support `primary` color change only. What I can imagine we could add is to extend support for the themes (we already have a selector for them). They could be dynamically loaded and injected i.e. via the ConfigMap to the pod.

The user would have to prepare a custom theme and upload it in order to use it in the Dashboard.
